### PR TITLE
Add new field for DbProject

### PR DESCRIPTION
**Pull request by djino97**
Added new field shortName for DbProject

### CODE REVIEW

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20200710073829_InitialCreate.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public partial class InitialCreate : Migration
         protected override void Up(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.CreateTable(
-                name: "Projects",
+                name: "Project",
```
Comment (lalagvana) [inline src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20200710073829_InitialCreate.cs:11]: Why Project, not Projects?

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/ProjectServiceDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -53,10 +56,10 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.HasKey("ProjectId", "FileId");
 
-                    b.ToTable("ProjectFile");
+                    b.ToTable("DbProjectFile");
```
Comment (lalagvana) [inline src/ProjectService.Data.Provider.MsSql.Ef/Migrations/ProjectServiceDbContextModelSnapshot.cs:59]: Why DbProjectFile?

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20200710073829_InitialCreate.cs (hunk 1 above)
Comment (djino97) [reply src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20200710073829_InitialCreate.cs]: Fixes

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/ProjectServiceDbContextModelSnapshot.cs (hunk 2 above)
Comment (djino97) [reply src/ProjectService.Data.Provider.MsSql.Ef/Migrations/ProjectServiceDbContextModelSnapshot.cs]: Fixed

**Review by DarkFelas — APPROVED** [review]

**Review by lalagvana — APPROVED** [review]

**Pull request merged**